Tasks/SonarQubePostTest/task.loc.json

```diff
@@ -12,7 +12,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 37
+    "Patch": 38
```

Comment:
Hi Bogdan, This file is task.loc.json which has  been generated on doing gulp. It seems some one after changing file SonarQubePostTest's task.json forget to checked in task.loc.json . Thats why it's get bumped up.